.yarn/plugins/engines-enforcer.cjs

```diff
@@ -0,0 +1,78 @@
+'use strict';
```

Comment:
This looks cool. Been having some node/yarn issues lately myself. Is it related to the task however? Maybe we can document how this is helpful